doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -679,7 +679,7 @@ Other samples
 -------------
 
 * Added the :ref:`coremark_sample` sample that demonstrates how to easily measure a performance of the supported SoCs by running the Embedded Microprocessor Benchmark Consortium (EEMBC) CoreMark benchmark.
-  Included support for the nRF52840 DK, nRF5340 DK, and nRF54L15 PDK.
+  Included support for the nRF52840 DK, nRF5340 DK, nRF54L15 PDK, and nRF54H20 DK.
```

Comment:
I think we should remove this sentence as support for specific DKs is marked in the table below

Reply:
@annwoj, @peknis, what do you think?

Reply:
Agree with @kapi-no. The supported DKs are mentioned in the beginning of the sample doc.

Reply:
Ok, I removed it.